hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java

```diff
@@ -1346,4 +1343,46 @@ public int getRange() {
     }
   }
 
+  /**
+   * Airflow specific error codes
+   */
+  public enum TensorBoardErrorCode implements RESTErrorCode {
+
+    TENSORBOARD_CLEANUP_ERROR(1, "Failed when deleting a running TensorBoard", Response.Status.INTERNAL_SERVER_ERROR),
+    TENSORBOARD_START_ERROR(2, "Failed to start TensorBoard", Response.Status.INTERNAL_SERVER_ERROR),
+    TENSORBOARD_FETCH_ERROR(3, "Error while fetching TensorBoard from database",
+            Response.Status.INTERNAL_SERVER_ERROR);
+
+    private Integer code;
+    private String message;
+    private Response.StatusType respStatus;
+    private final int range = 300000;
```

Comment:
@robzor92 should be `28000`, not `300000`

Reply:
Fixed